### PR TITLE
Update transaction api request details

### PR DESCRIPTION
**Pull request by lp-gdoshi**

### CODE REVIEW

File: pages/Documents/Reporting/OutboundReportingAPI/overview.md [hunk 1]
```diff
@@ -494,6 +515,165 @@ Click [**Transaction**](https://proactive-messaging.z1.fs.liveperson.com/api/api
...
+            "deliveredTime": "2021-02-17T23:00:18.533Z",
+            "eligibleTime": null,
+            "errorCode": null,
+            "errorMessage": null,
+            "errorSource": null,
+            "failedTime": null,
+            "inviteTime": "2021-02-17T22:57:13.480Z",
+            "optInsTime": null,
+            "optOutsTime": null,
+            "proactiveCampaignId": "camapign125",
```
Comment (jschatzman-lp) [inline pages/Documents/Reporting/OutboundReportingAPI/overview.md:634]: maybe fix spelling

File: pages/Documents/Reporting/OutboundReportingAPI/overview.md [hunk 2]
```diff
@@ -494,6 +515,165 @@ Click [**Transaction**](https://proactive-messaging.z1.fs.liveperson.com/api/api
...
+            "deliveredTime": "2021-02-17T23:00:18.533Z",
+            "eligibleTime": null,
+            "errorCode": "400",
+            "errorMessage": "Message failed to send",
+            "errorSource": "prmsg",
+            "failedTime": "2021-02-17T22:55:14.543Z",
+            "inviteTime": "2021-02-17T22:55:14.543Z",
+            "optInsTime": null,
+            "optOutsTime": null,
+            "proactiveCampaignId": "camapign124",
```
Comment (jschatzman-lp) [inline pages/Documents/Reporting/OutboundReportingAPI/overview.md:663]: maybe fix spelling

**Review by jschatzman-lp — APPROVED** [review]

File: pages/Documents/Reporting/OutboundReportingAPI/overview.md [hunk 3]
```diff
@@ -346,11 +346,16 @@ Click [**Campaign**](https://proactive-messaging.z1.fs.liveperson.com/api/api-do
...
+API for account level transactions. 
 
-**1. Get details for transactions**
+There are two versions of transaction API's.
 
-Click [**Transaction**](https://proactive-messaging.z1.fs.liveperson.com/api/api-docs/?api=reporting#/Transaction/transactions) to go through API spec to get started.
+| Version | Description  |
+| :--- | :--- |
+| Version 1.0 (Deprecated) | Existing version of transaction API with filtering on channels and skills |
+| Version 2.0 (Recommended) | New version with simplified way for pagination and with additional filtering on message status and transaction id's along with channles and skills |
```
Comment (jschatzman-lp) [inline pages/Documents/Reporting/OutboundReportingAPI/overview.md:356]: fix channels spelling

File: pages/Documents/Reporting/OutboundReportingAPI/overview.md [hunk 4]
```diff
@@ -494,6 +515,165 @@ Click [**Transaction**](https://proactive-messaging.z1.fs.liveperson.com/api/api
...
+| Header | Description | Value/Example |
+| :--- | :--- | :--- |
+| Content-Type | Used to indicate the media type of the resource | application/json |
+| Authorization | [OAuth 2.0](https://developers.liveperson.com/connector-api-send-api-authorization-and-authentication.html#get-appjwt) or [OAuth 1.0](https://developers.liveperson.com/retrieve-api-keys-create-a-new-api-key.html) (Section 8) or LE Bearer token | |
+
+**Request Body**
+
+| Key | Description | Values/Examples |
+| :--- | :--- | :--- |
+| offset | The offset specifies from which record to retrieve the data | Default = 0 |
```
Comment (jschatzman-lp) [inline pages/Documents/Reporting/OutboundReportingAPI/overview.md:553]: mention that it is ordered by transaction id

Comment (lp-gdoshi) [reply pages/Documents/Reporting/OutboundReportingAPI/overview.md:553]: I dont think this is required. Ultimately user will get data sorted by transaction hour.

File: pages/Documents/Reporting/OutboundReportingAPI/overview.md (hunk 1 above)
Comment (lp-gdoshi) [reply pages/Documents/Reporting/OutboundReportingAPI/overview.md]: Done

File: pages/Documents/Reporting/OutboundReportingAPI/overview.md (hunk 2 above)
Comment (lp-gdoshi) [reply pages/Documents/Reporting/OutboundReportingAPI/overview.md]: Done

File: pages/Documents/Reporting/OutboundReportingAPI/overview.md (hunk 3 above)
Comment (lp-gdoshi) [reply pages/Documents/Reporting/OutboundReportingAPI/overview.md]: Done

**Pull request merged**